Kitodo-API/src/main/java/org/kitodo/api/ugh/MetadataTypeInterface.java

```diff
@@ -0,0 +1,155 @@
+ * Information which they share are information about their occurrences; each
+ * structure entity can only have a single title.
+ *
+ * <p>
+ * MetadataType objects can occur in two different ways:
+ * <ul>
+ * <li>globally
+ * <li>locally
+ * </ul>
+ *
+ * <b>Global</b> {@code MetadataType} objects can be retrieved from the
+ * {@code Prefs} object by giving the internal name. Some of the information of
+ * a MetadataType object depends on the context in which it is used. Context
+ * means it depends on the {@code DocStructType} object, in which a MetadataType
+ * object is used. When adding a {@code MetadataType} object to a
+ * {@code DocStructType} object, an internal copy is created and stored with the
+ * {@code DocStructType} object. This copy is called <b>local</b> and may store
+ * information about its occurrences in this special {@code DocStructType}
+ * object. The {@code DocStructType} class contains methods to retrieve local
+ * {@code MetadataType} objects from global ones.
+ *
+ * <p>
+ * {@code MetadataType} objects are used to create new {@code MetadataType}
+ * objects. They are the only parameter in the constructor of the
+ * {@code MetadataType} object.
+ */
+public interface MetadataTypeInterface {
+
+    /**
+     * Returns all display labels of this meta-data type.
+     *
+     * @return a map of display labels of this meta-data type
+     */
+    Map<String, String> getAllLanguages();
+
+    /**
+     * Returns whether the meta-data type is a person type or not.
+     *
+     * @return whether the meta-data type is a person type
+     */
+    boolean getIsPerson();
+
+    /**
+     * Returns the display label of this meta-data group type in a given
+     * language.
+     *
+     * @param language
+     *            language of the display label to return
+     * @return the display label in the given language
+     */
+    String getLanguage(String language);
+
+    /**
+     * Returns the internal name (that is, the ID) of this meta-data group.
+     *
+     * @return the ID of this
+     */
+    String getName();
+
+    /**
+     * Returns the display label of this meta-data group type in a given
+     * language.
+     *
+     * @param language
+     *            language of the display label to return
+     * @return the display label in the given language
+     */
+    default String getNameByLanguage(String language) {
+        return getLanguage(language);
+    }
+
+    /**
+     * Gets the minimum required/maximum allowed quantity for this meta-data
+     * group type.
+     *
+     * @return one of "1m", "1o", "+", or "*"
+     */
+    String getNum();
+
+    /**
+     * Sets all display labels for this meta-data type.
+     *
+     * @param allLanguages
+     *            the map of display labels of this meta-data type
+     * @return always {@code true}. The return value is never used.
+     */
+    boolean setAllLanguages(HashMap<String, String> labels);
```

Comment:
1) See my comment to  `setLocation`.
2) Please adjust the JavaDoc or method signature as they are different.